opentraveldata/optd_por_best_known_so_far.csv

```diff
@@ -12803,7 +12803,8 @@ SHA-C-1796236^SHA^31.22222^121.45806^SHA^
 SHB-A-6300326^SHB^43.5775^144.96^SHB^
 SHB-C-2129034^SHB^43.54849^144.97181^SHB^
 SHC-CA-7731491^SHC^14.08^38.28^SHC^
-SHD-CA-4785295^SHD^38.27^-78.9^SHD^
+SHD-A-4785295^SHD^38.26292^-78.90202^SHD^
+SHD-C-4785243^SHD^38.48512^-78.62502^SHD^
```

Comment:
- Do not hesitate, on Geonames:
 - To fix the feature name
 - To add a few alternate names, to be found on the corresponding Wikipedia page (e.g., http://en.wikipedia.org/wiki/Shenandoah%2C_Virginia)
 - To add some missing administrative levels (administrative level 1 at least). You can figure out the correct one by looking at other POR around, where those administrative levels are set
- And do not hesitate to check that the geographical coordinates are the same in OpenTravelData and in Geonames